docs/FAQ.md

```diff
@@ -223,6 +223,114 @@ Questions about VictoriaMetrics can be asked via the following channels:
+
+
+### Is it possible to delete a tenant by deleting its metrics?  
+
+Yes, it’s possible. Also we have this [issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1111).
+
+
+### How to set a memory limit for vmstorage ? Cluster/SingleNode version
+
+VictoriaMetrics provides flags to control the size of internal buffers and caches. But these are soft limits and the total process memory usage may exceed it. Hard limits may be enforced only by OS ([cgroups](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/sec-relationships_between_subsystems_hierarchies_control_groups_and_tasks)) or virtualization tools ([docker](https://docs.docker.com/config/containers/resource_constraints), [k8s](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers)).
+
+
+### Can I use VictoriaMetrics instead of an all Prometheus Stack (Prometheus, Alertmanager, PushGateway) or are there some things that can't be replaced?
+
+Yes, you can replace your entire Prometheus stack with VictoriaMetrics
+
+
+### I have two Prometheus replicas in my k8s cluster that write data into VictoriaMetrics and therefore see dual data from node_exporter’s in my Grafana dashboard. I have tried to use  `-dedup.minScrapeInterval: 10s`  parameter for vmagent https://docs.victoriametrics.com/#deduplication but my Pod was disconnected because it didn’t know this flag.
+
+`-dedup.minScrapeInterval` supported by single node version and for cluster version it supported by vmselect/vmstorage services so you need to set this flag for them.
+
+
+### How can I build VictoriaMetrics on FreeBSD?
+
+To build VictoriaMetrics on FreeBSD you need to install Golang in OS and then run gmake victoria-metrics-pure or install it from FreeBSD Ports - see [this link](https://www.freebsd.org/cgi/ports.cgi?query=victoria&stype=all) .
+
+
+### What is the rule of thumb for vminsert replication factor? What if I have 8 vmstorage pods?
+
+E.g. replication factor 3 means that you have your initial data copied 3 times and these copies are spread across 8 nodes. Every node will therefore have 3/8 of the initial data.
+
+
+### Can I make a query to different tenants at the same time? 
+
+No. You can only do this if you will run the query through [promxy](https://github.com/jacksontj/promxy).
+
+
+### How does vmselect know in which storage to find the entire dataset?  
+
+`vmselect` queries all the `vmstorage` instances. See [these docs](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#replication-and-data-safety) for details.
+
+
+### Is there a way to use vm to identify the top 10 slow queries or non-performing queries being made to VictoriaMetrics?
```

Comment:
would be great to be consistent in naming and case:
vm => VictoriaMetrics
multitenant => multi-tenant
etc.